cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -167,10 +167,7 @@ func getMsgKey(obj interface{}) (string, error) {
 	msg := obj.(*beehiveModel.Message)
 
 	if msg.GetGroup() == edgeconst.GroupResource {
-		resourceType, _ := messagelayer.GetResourceType(*msg)
-		resourceNamespace, _ := messagelayer.GetNamespace(*msg)
-		resourceName, _ := messagelayer.GetResourceName(*msg)
-		return strings.Join([]string{resourceType, resourceNamespace, resourceName}, "/"), nil
+		return GetMessageUID(*msg)
```

Comment:
Will the `getMsgKey` method involve the parsing of the upper-level method, if it is changed to UID

Reply:
No, we only have two place use the key func, one is when we add the message to the queue, the another is that we pass the  keyFunc `getMsgKey` to the messageStore, so the internal change will not affect  upper-level method

Reply:
thanks for your explanation.